molecule/molecule-testing/src/main/kotlin/app/cash/molecule/testing/moleculeTesting.kt

```diff
@@ -52,6 +52,11 @@ fun <T> testMolecule(
 ) = testMolecule(body, timeoutMs.milliseconds, validate)
 
 @ExperimentalCoroutinesApi
+@Deprecated(
+  """Use moleculeFlow or launchMolecule with an 'Immediate' clock and then use Turbine to test.
```

Comment:
Is `launchMolecule` + `Turbine` a legitimate combination that can be made? Would be a BYO plumbing sort of deal?

Reply:
Both APIs have the option of using the immediate clock. Since `StateFlow` is still a `Flow` you can still test it with Turbine.

Reply:
Right. I forgot the signatures.

Reply:
I would recommend testing with `moleculeFlow` over `launchMolecule` due to [stateflow testing being annoying](https://www.billjings.net/posts/title/testing-stateflow-is-annoying/)